src/app/collective-rewards/rewards/builders/EstimatedRewards.tsx

```diff
@@ -53,34 +52,37 @@ const TokenRewards: FC<TokenRewardsProps> = ({ gauge, token: { id, symbol }, cur
     isLoading: rewardSharesLoading,
     error: rewardSharesError,
   } = useGetRewardShares(gauge)
+  const {
+    data: rewardPercentage,
+    isLoading: rewardPercentageLoading,
+    error: rewardPercentageError,
+  } = useGetRewardPercentageToApply(builder)
 
-  const error = rewardsError ?? totalPotentialRewardsError ?? rewardSharesError
+  const error = rewardsError ?? totalPotentialRewardsError ?? rewardSharesError ?? rewardPercentageError
   useHandleErrors({ error, title: 'Error loading estimated rewards' })
 
   const { prices } = usePricesContext()
 
+  const precision = parseUnits('1', 18)
+  const rewardsAmount = rewardPercentage ? (rewards * rewardPercentage) / precision : 0n
```

Comment:
If `rewardPercentage` is 0, then the result is 0, that is fine
```suggestion
  const rewardsAmount = (rewards * rewardPercentage) / precision : 0n
```